### DISCUSSION TITLE
Double NATed clients can't connect to a call in Talk

### DISCUSSION THREAD

**Discussion by Byter3** [nextcloud/all-in-one#3519] [Questions] [answered]
Nextcloud AIO v7.2.1
Talk 17.1.1
Double NATed clients can't connect to a call in Talk.

**Comment by szaimen** [comment, marked as answer]
Hi, this should be fixed with AIO 7.41 Beta. See https://github.com/nextcloud/all-in-one#how-to-switch-the-channel

**Reply by Byter3** [reply]
This also?
Clients can't reconnect to a call in Talk after switching networks ie: disconnecting from wifi and switching to mobile data.

**Reply by szaimen** [reply]
I am not sure. This rather sounds like a bug in the mobile apps honestly.

**Reply by Byter3** [reply]
Thank you!
Will open a bug report there than
